### DISCUSSION TITLE
Tweepy and Twitter API won't be able to verify credentials for app.

### DISCUSSION THREAD

**Discussion by MinimumADHD** [tweepy/tweepy#1859] [Questions] [answered]
When I try to make any kind of app, tried even using different type of OA**uthHandler, Client and those** but none eventually seem to work and PyCharm terminal keeps printing "Bad"

```py
import tweepy

#i eventually put the token but i'm hiding it in there

auth = tweepy.Client(BEARED_TOKEN)

api = tweepy.API(auth)

try:
    api.verify_credentials()
    print("Working!")
except:
    print("Bad")
```
![image](https://user-images.githubusercontent.com/91323541/162639919-3ee5f302-3fe1-4f98-8a3b-887f890ecf88.png)

**Comment by Harmon758** [comment, marked as answer]
This isn't how you [authenticate](https://tweepy.readthedocs.io/en/v4.8.0/authentication.html) with [`API`](https://tweepy.readthedocs.io/en/v4.8.0/api.html) or [`Client`](https://tweepy.readthedocs.io/en/v4.8.0/client.html).
In the future, you need to provide the full unobfuscated traceback when asking for help.
